test/user.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+process.env.NODE_ENV = 'testing';
+
+let mongoose = require("mongoose");
+let User = require('../server/api/user/model');
+
+//Require the dev-dependencies
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../index');
+let should = chai.should();
+
+chai.use(chaiHttp);
+
+let user = {
+    username: 'admin',
+    password: 'changeit'
+    };
+
+let jwt = '';
+
+describe('User', () => {
```

Comment:
We should add 'after' function to ensure data always be cleaned.

We should do like this:

cleanData = (done) => {
User.remove({}, (err) => {
done();
})
};

before(cleanData);
after(cleanData);